dockerfiles/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM opensuse:42.1
+
+RUN zypper install -y bash \
+        bison \
+        bzip2 \
+        cmake \
+        curl \
+        flex \
+        gcc-c++ \
+        gettext \
+        git \
+        libzip2 \
+        glib2-devel \
+        pthread-stubs-devel \
+        readline-devel \
+        libopenssl-devel \
+        libSM6 \
+        make \
+        python-numpy \
+        ncurses-devel \
+        java-1_8_0-openjdk-headless \
+        openssl \
+        patch \
+        python-devel \
+        python-setuptools \
+        zlib \
+        zlib-devel \
+        tar \
+        libXext6 \
+        libbz2-devel \
+        libXrender1 \
+        gcc48-fortran \
+        which
+
+ARG LSST_STACK_DIR=/lsst/stack
+ARG LSST_USER=desc
+ARG LSST_GROUP=desc
+ARG LSST_WORKSPACE=/desc
+
+ARG EUPS_HOME=$LSST_STACK_DIR/.eups
+ARG EUPS_PRODUCT=lsst_distrib
+ARG EUPS_TAG=v13_0
+ARG INSTALL_MINICONDA=Y
+
+RUN mkdir -p $LSST_STACK_DIR
+RUN mkdir -p $EUPS_HOME
+RUN mkdir -p $LSST_WORKSPACE
+
+WORKDIR $LSST_STACK_DIR
+
+RUN echo "Environment: \n" && env | sort
+
+# Explicitly setup user and user groups
+RUN groupadd $LSST_GROUP
+RUN useradd -m -g $LSST_USER $LSST_GROUP
+RUN usermod -s /bin/bash $LSST_USER
+
+# Install stack
+# If $EUPS_TAG exists, prepend with a "-t"
+RUN echo "Executing: eups distrib install $EUPS_TAG_OPT $EUPS_PRODUCT $EUPS_PRODUCT_VERSION" && \
+    curl -OL https://raw.githubusercontent.com/lsst/lsst/13.0/scripts/newinstall.sh && \
```

Comment:
I think we want -> `https://raw.githubusercontent.com/lsst/lsst/master/scripts/newinstall.sh`

I know this is based on master, but lsst/lsst does get fixes based on infrastructure evolution, so it might be alright to go ahead and base this part off of master.